README.md

```diff
@@ -77,7 +79,7 @@ Installing on macOS:
 
 - Install dependencies using Homebrew:
   ```
-  brew install autoconf automake libtool bdw-gc boost bison pkg-config
+  brew install autoconf automake libtool bdw-gc boost bison pkg-config protobuf
```

Comment:
Unfortunately Ubuntu16.04's default protobuf is v2.6.1, which configure.ac apparently thinks is too old, and there's not an easy way to get a newer version on Ubuntu -- need to download the package from github and build?